pkgs/by-name/el/ell/package.nix

```diff
@@ -36,6 +36,11 @@ stdenv.mkDerivation rec {
   # Runs multiple dbus instances on the same port failing the bind.
   enableParallelChecking = false;
 
+  # 'unit/test-hwdb' fails in the sandbox as it relies on
+  # '/etc/udev/hwdb.bin' file presence in the sandbox. `nixpkgs` does
+  # not provide it today in any form. Let's skip the test.
+  env.XFAIL_TESTS = "unit/test-hwdb";
+
```

Comment:
https://github.com/NixOS/nixpkgs/blob/f680a9ecd4fa48fc00d3a3cd6babb44cc48a4408/nixos/modules/services/hardware/udev.nix#L143-L163

but that's a derivation produced by a NixOS module. So I suppose it's not worth the effort to try to obtain it just for this test. Let's hope it's tested upstream somehow.